templates/base.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+    {% block title %}
+        <title>Taxi service</title>
+    {% endblock %}
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/style.css' %}">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.2/dist/css/bootstrap.min.css" integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N" crossorigin="anonymous">
+</head>
+<body>
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-sm-3">
+          {% block sidebar %}
+            {% include "includes/sidebar.html" %}
+          {% endblock %}
+      </div>
+      <div class="col-sm-9">
+          {% block content %}{% endblock %}
+      </div>
+    </div>
+  </div>
+</body>
+</html>
```

Comment:
put empty lines at the end of each HTML file

Reply:
Fixed.